protocol/x/rewards/testdata/expected_default_genesis.json

```diff
@@ -0,0 +1,9 @@
+{
+  "params": {
+    "treasury_account":"rewards_treasury",
+    "denom":"dv4tnt",
```

Comment:
This needs to be updated to `adv4tnt` due to this [PR](https://github.com/dydxprotocol/v4-chain/pull/454)